recipes/master_config_post.rb

```diff
@@ -29,41 +29,41 @@
 end
 
 execute 'Import Openshift Hosted Examples' do
-  command "#{node['cookbook-openshift3']['openshift_common_client_binary']} create -f #{node['cookbook-openshift3']['openshift_common_hosted_base']} --recursive --config=admin.kubeconfig -n openshift || #{node['cookbook-openshift3']['openshift_common_client_binary']} replace -f #{node['cookbook-openshift3']['openshift_common_hosted_base']} --recursive --config=admin.kubeconfig -n openshift"
+  command "#{node['cookbook-openshift3']['openshift_common_client_binary']} create -f #{node['cookbook-openshift3']['openshift_common_hosted_base']} --config=admin.kubeconfig -n openshift || #{node['cookbook-openshift3']['openshift_common_client_binary']} replace -f #{node['cookbook-openshift3']['openshift_common_hosted_base']} --config=admin.kubeconfig -n openshift"
```

Comment:
if recursive flag is removed (v1.2.x compatibility) how are all loaded created from the template directory? IIRC "oc create -f" only accepts a single argument which is a file, unless recursive specified